.github/workflows/coderabbit-auto-fix.yml

```
@@ -1,90 +1,105 @@
 
-      - name: Checkout repo
+      - name: Checkout PR branch
         uses: actions/checkout@v4
         with:
-          ref: ${{ github.event.pull_request.head.sha }}
+          ref: ${{ github.event.pull_request.head.ref }}
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Extract CodeRabbit comment
+        id: comment
+        run: |
```

actionlint (1.7.11)

[error] "github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)